packages/connectivity/android/src/main/AndroidManifest.xml

```diff
@@ -0,0 +1,7 @@
+<manifest xmlns:android="http://schemas.android.com/apk/res/android"
+  package="io.flutter.plugins.connectivity"
+  android:versionCode="1"
+  android:versionName="0.0.1">
+
+  <uses-sdk android:minSdkVersion="16" android:targetSdkVersion="21" />
+</manifest>
```

Comment:
Do we need to request a specific android permission in order to be able to access `CONNECTIVITY_SERVICE`?